nullaway/src/main/java/com/uber/nullaway/handlers/OptionalEmptinessHandler.java

```diff
@@ -164,17 +165,18 @@ private boolean isOptionalContentNullable(
   }
 
   @Override
-  public boolean includeApInfoInSavedContext(AccessPath accessPath, VisitorState state) {
-
-    if (accessPath.getElements().size() == 1) {
-      final Element e = accessPath.getRoot();
-      if (e != null) {
-        return e.getKind().equals(ElementKind.LOCAL_VARIABLE)
-            && accessPath.getElements().get(0).getJavaElement()
-                instanceof OptionalContentVariableElement;
+  public Predicate<AccessPath> getAccessPathPredicateForNestedMethod(
+      TreePath path, VisitorState state) {
+    return ap -> {
+      if (ap.getElements().size() == 1) {
+        final Element e = ap.getRoot();
+        if (e != null) {
+          return e.getKind().equals(ElementKind.LOCAL_VARIABLE)
+              && ap.getElements().get(0).getJavaElement() instanceof OptionalContentVariableElement;
+        }
       }
-    }
-    return false;
+      return false;
+    };
```

Comment:
Unrelated to this PR, but... are we missing unit tests for this handler? Or is Codecov wrong?

Reply:
We are in fact missing coverage!  It turns out that #678 made the pseudo-field simulating contents of an `Optional` final, which means we already (safely) propagate nullability facts about that pseudo-field into lambdas, which means we no longer need this handler method!  I'll go ahead and delete this method in a separate PR.  Nice to have the test coverage exposed!